std/typecons.d

```diff
@@ -6485,8 +6488,11 @@ if (!is(T == class) && !(is(T == interface)))
         /**
            Makes sure the payload was properly initialized. Such a
            call is typically inserted before using the payload.
+
+           This function is unavailable if `T.this()` is annotated with
+           `@disable`.
         */
-        void ensureInitialized()
+        void ensureInitialized()()
```

Comment:
I'm not sure if making this a zero-args template is the cleanest solution for user-facing code. Would it be better to use a `static if`?